Deliverance/admin/components/Newsletter/edit.xml

```diff
@@ -5,6 +5,14 @@
 	<widget class="SwatFrame" id="edit_frame">
 		<property name="title" translatable="yes">Newsletter</property>
 		<widget class="SwatForm" id="edit_form">
+			<widget class="SwatFormField" id="template_field">
+				<property name="title" translatable="yes">Template</property>
+				<property name="visible" type="boolean">false</property>
+				<widget class="SwatRadioList" id="template">
+					<property name="required" type="boolean">true</property>
+					<property name="show_blank" type="boolean">true</property>
```

Comment:
This line does nothing. This property is always set to `false` in `SwatRadioList::__construct()`.